apps/backend/src/groups/groups.service.ts

```diff
@@ -0,0 +1,60 @@
+import { Injectable } from '@nestjs/common';
+import { Group, Prisma } from '@prisma/client';
+import { PrismaService } from 'nestjs-prisma';
+
+@Injectable()
+export class GroupsService {
+  constructor(private readonly prisma: PrismaService) {}
+
+  async create(data: Prisma.GroupCreateInput): Promise<Group> {
+    return await this.prisma.group.create({ data });
+  }
+
+  async findAll(): Promise<Group[]> {
+    return this.prisma.group.findMany();
+  }
+
+  async findOne(id: number): Promise<Group> {
+    const group = await this.prisma.group.findUnique({ where: { id: id } });
+    if (!group) {
+      throw new Error(`Group with ID ${id} not found`);
```

Comment:
A nest-nek van beépített hibája a különböző hibákra, pl. `NotFoundException`. Ezek azért jobbak, mert automatikusan a megfelelő hibakódot is visszaadják.
A hibakódokról bővebben: [HTTP Állapotkódok](https://hu.wikipedia.org/wiki/HTTP-%C3%A1llapotk%C3%B3dok)